phpunit.xml.dist

```diff
@@ -13,7 +13,7 @@
 
         <testsuite name="phpcr">
             <directory>./Tests/Functional</directory>
-            <exclude>./Tests/Functional/Doctrine/Orm</exclude>
+            <exclude>./Tests/Functional/Doctrine/Phpcr</exclude>
```

Comment:
No, the PHPCR test suite executes everything in `./Tests/Functional` except from `./Tests/Functional/Doctrine/Orm`. The ORM testsuite only executes `./Tests/Functional/Doctrine/Orm`.